namui/src/namui/utils/simple_error_impl.rs

```diff
@@ -0,0 +1,11 @@
+#[macro_export]
+macro_rules! simple_error_impl {
```

Comment:
This is additional macro which provide making error impl for struct very easily.